### DISCUSSION TITLE
The speechrecogination dosen't work

### DISCUSSION THREAD

**Discussion by Youssefmb** [JamesBrill/react-speech-recognition#231] [Q&A]
`const {
        transcript,
        listening,
        browserSupportsSpeechRecognition
    } = useSpeechRecognition();

    if (!browserSupportsSpeechRecognition) {
        return <span>Browser doesn't support speech recognition.</span>;
    }
`

`
<>
                    <div>
                        <TextArea
                            disabled
                            value={transcript}
                            className='bg-white position-absolute ml-4'
                            style={{ 'width': '15rem', 'top': '15rem', 'height': '50%' }}
                        >
                        </TextArea>
                    </div>
                    <div className='position-absolute ml-4' style={{ 'top': '40rem' }}>

                        <p className='text-white'>AI detector : {listening ? 'on' : 'off'}</p>
                        <Button className='mr-1' onClick={SpeechRecognition.startListening({ continuous: true })}>Start</Button>
                        <Button className='mr-1' onClick={SpeechRecognition.stopListening}>Stop</Button>
                    </div>
                </>
`

here the transcript is not appair in the textArea or any other balise , i think it dosen't work , why?

**Comment by Youssefmb** [comment]
i solve it , i just change the default microphone of the device.

**Comment by ashw1nshenoy** [comment]
Facing the same issue can you help me out

**Reply by Youssefmb** [reply]
how can i help you ? tell me more details.
